src/main/java/com/redhat/digkins/services/CreateJobService.java

```diff
@@ -27,9 +26,9 @@ public CreateJobService(JenkinsServer jenkins) {
   /**
    * Create new digger job on jenkins platform
    *
-   * @param name      - job name that can be used later to reference job
-   * @param gitRepo   - git repository url (full git repository url. e.g git@github.com:digger/helloworld.git
-   * @param gitBranch - git repository branch (default branch used to checkout source code)
+   * @param name      job name that can be used later to reference job
+   * @param gitRepo   git repository url (full git repository url. e.g git@github.com:digger/helloworld.git
+   * @param gitBranch git repository branch (default branch used to checkout source code)
    */
   public void create(String name, String gitRepo, String gitBranch) throws IOException {
     JtwigTemplate template = JtwigTemplate.classpathTemplate("templates/job.xml");
```

Comment:
I strongly recommend to extract hardcoded strings into a constant field, this way it can be found, checked and modified easier when needed.